docs/plasmapy_sphinx/__init__.py

```diff
@@ -1,7 +1,6 @@
 """
-This `Sphinx <https://www.sphinx-doc.org/en/master/>`_ extension package contains
-custom documentation functionality used to document
-`PlasmaPy's packages <https://github.com/PlasmaPy>`_ .
+This Sphinx_ extension package contains custom documentation functionality
+used to document `PlasmaPy's packages <https://github.com/PlasmaPy>`_ .
```

Comment:
Can you revert this back?  I only say that so the Sphinx link is not broken when I spin this off into it's own package.  After I spin the package off I'll create a common links for that package.